nextstrain_profiles/nextstrain/builds.yaml

```diff
@@ -1,3 +1,5 @@
+auspice_json_prefix: ncov_gisaid
```

Comment:
What's the best recommendation here for nextstrain.org redirects?

We can probably hardcode top-level redirects `/ncov/{global,africa,...}` → `/ncov/gisaid/{global,africa,...}`

Do we want to preserve all the old (timestamped) URLs, or change the filenames & add redirects accordingly? (I don't think it's worth it.) Going forward, newly produced timestamped datasets will be of the `ncov/gisaid/:region/YYYY-MM-DD` format, correct?

cc @eharkins

Reply:
Yep. We'll want to hardcode top-level redirects.

I don't recall discussion about redirects or not for the dated URLs, but I think it would be ok to leave them as-is. We can always move them and add redirects later if we change our mind.

The date stamping suffixes will remain the same and happen for both GISAID and Open builds.